app/package.json

```diff
@@ -23,6 +23,9 @@
     "storybook": "start-storybook -s ."
   },
   "xo": {
+    "ignores": [
+      "cypress"
+    ],
```

Comment:
That's a bit excessive. We can override the xo config and disable the one rule that should be disabled

Reply:
See https://github.com/xojs/xo#config-overrides